aesara/link/numba/dispatch/elemwise.py

```diff
@@ -195,6 +209,33 @@ def careduce_axis(x):
     return careduce_axis
 
 
+def create_elemwise_reducer(ndim, axis, reduce_fn):
+    reduce_elemwise_fn_name = "_reduce_elemwise"
+    global_env = {}
+    res_cells = ""
+    arr_cells = ""
+    count = 0
+
+    for i in range(ndim):
+        if i == axis:
+            arr_cells += "[i]"
+        else:
+            res_cells += f"[idx_arr[{count}]]"
+            arr_cells += f"[idx_arr[{count}]]"
+            count = count + 1
+    # TODO:Apply reduce fns over here
+    reduce_elemwise_def_src = f"""
+def {reduce_elemwise_fn_name}(res, x, axis, idx_arr):
+    for i in range(x.shape[axis]):
+        res{res_cells} = res{res_cells} + x{arr_cells}
```

Comment:
One question with his approach is how exactly should we perform the elementary operation over here. Having a `np.add` for instance would add an extra overhead over here, no?